src/model.ts

```diff
@@ -142,6 +142,8 @@ abstract class $Node {
 
   isSynthetic(): this is this & { sourceMap: undefined } { return !this.sourceMap }
 
+  hasProblems(): boolean { return !!this.problems && this.problems.length > 0 }
```

Comment:
No es importante, pero esto también podría escribirse: `this.problems?.length > 0`

Reply:
Tenés razón, entre Kotlin y TS se me van pisando lo que puedo hacer...

Todavía lo tengo en draft, le voy a agregar el extension method isEmpty y el fix que me sugeriste para el validador de only call to super y te aviso...